packages/tpetra/core/src/Tpetra_CrsMatrix_def.hpp

```diff
@@ -5927,42 +5927,36 @@ namespace Tpetra {
         Distributor& dist) const
   {
```

Comment:
I agree.  I focused only on using PackTraits this go around.  Before merging, Kokkos views should be used.